x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -27,6 +26,7 @@ import type { IRuleDataClient } from '@kbn/rule-registry-plugin/server';
 import { Dataset } from '@kbn/rule-registry-plugin/server';
 import type { ListPluginSetup } from '@kbn/lists-plugin/server';
 import type { ILicense } from '@kbn/licensing-plugin/server';
+import LRU from '../node_modules/lru-cache';
```

Comment:
is this change intentional?

Reply:
It is change automatically done by `node scripts/precommit_hook.js --fix`, which is strange why.

Reply:
Could be an error with `main`. Let me update it.

Reply:
Still the error is there but I removed it and posted query here: https://elastic.slack.com/archives/C5UDAFZQU/p1681809131416439